notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>I noticed that in the Forecasting MP, the function batch_predict takes four parameters instead of 5: https://cloud.google.com/vertex-ai/docs/tabular-data/forecasting/get-predictions. What is "instances_format"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>instances_format is to specify the batch prediction input format.</p>

Reply:
<p>So, could you please clarify: do Tabular Workflow for Forecasting and the Forecasting MP use different functions for making predictions? Or is "instances_format" an optional parameter? </p>

Reply:
<p>Yes, this is the same function, this argument is optional, the default value is <code>jsonl</code> : https://github.com/geraint0923/python-aiplatform/blob/master/google/cloud/aiplatform/models.py#L1809 </p>